eng/scripts/CodeChecks.ps1

```diff
@@ -55,6 +55,11 @@ function Invoke-Block([scriptblock]$cmd) {
 }
 
 try {
+    Write-Host "Initialize npx cache"
+    Invoke-Block {
+        & npx autorest --version
```

Comment:
How does initialize it? Does npx leave the pacakge in the node_modules cache?

Reply:
npx uses its own global cache that sits inside the npm cache folder in AppData

Reply:
So we are just seeding that cache so the next calls to npx use it? What if there are any other package updates in the dependency chain will that still cause conflict?